### ISSUE TITLE
Icon(s) Request: Food

### ISSUE THREAD

**Issue by itsezc** [Remix-Design/RemixIcon#85]
## Icon Request

* Icon name: Food
* Tags of this icon: burger, drink, alcohol etc.
* Use case:  For food related apps

Screenshots of some from Fontawesome:

![image](https://user-images.githubusercontent.com/33750251/64995595-8b671b00-d8d3-11e9-9d10-283e9fe4e0cd.png)

![image](https://user-images.githubusercontent.com/33750251/64995631-acc80700-d8d3-11e9-80a9-af060e109977.png)

Even if its not all of these, we could have a few i.e. meat, burger, cheese, wheat etc. to differentiate between allergies etc.

**Comment by xiaochunjimmy** [comment]
Thank you for your advice. We'll deal with some of them in the feature versions

**Comment by iamrishishah** [comment]
@itsezc super request. @xiaochunjimmy it's really needed. Thanks in advance.

**Comment by itsezc** [comment]
@xiaochunjimmy I'd also request the following additional "Food" icons

Chef:

![image](https://user-images.githubusercontent.com/33750251/79042625-08166980-7c17-11ea-88de-ee055f3efd94.png)

Spatula:

![image](https://user-images.githubusercontent.com/33750251/79042622-fb921100-7c16-11ea-9bba-d74e84ee2281.png)

**Comment by nimabrownlee** [comment]
Hey,
Is this being worked on?

**Comment by itsezc** [comment]
@nimabrownlee Yes, as it is listed as icon request. I am sure @xiaochunjimmy will get to them asap.

**Comment by ngqq** [comment]
Would also like to see more in the food category for things like a takeaway icon (box with chopsticks) or restaurant

**Comment by sNiXx** [comment]
That would be fantastic to have! It would certainly help if there was at least some general icon for the food category (e.g. an apple)

**Comment by whoismuktar** [comment]
2023, any update on food related icon?

**Comment by bedimcode** [comment]
I was working on a design and I noticed that there aren't many food icons available.

Missing: 
- **Hamburger**
- **Pizza**
- **Sushi**
- **Salads**
- **Sandwiches**
- **And more**

I hope they update the icon library in the food category this year. Regards.
